rgo/component.py

```diff
@@ -78,35 +77,37 @@ def make_srpm(self, tmpdir):
             patches = PatchesAction.keep
 
         # extract spec file from specified branch and save it in temporary location
+        import rpm
```

Comment:
Since the caues for this (travis) was eliminated, maybe it could be a global import now?

Reply:
Changed.